components/CKEditor/ELNEditor.js

```diff
@@ -92,10 +90,27 @@ const ELNEditor = ({
       setPresenceListElement(node);
     }
   }, []);
-
   const currentUserAccess = getUserNoteAccess({ user, notePerms, userOrgs });
   const noteIdLength = `${currentNote.id}`.length;
   const channelId = `${orgSlug.slice(0, 59 - noteIdLength)}-${currentNote.id}`;
+  const parsedNoteTitle = unescapeHtmlString(
+    editorInstance?.plugins?.get("Title")?.getTitle() ?? "Untitled"
+  );
+
+  useEffect(() => {
+    if (
+      !(isNullOrUndefined(typeof window) && isNullOrUndefined(typeof document))
```

Comment:
This is fine but I think just checking `if (process.browser)` will do the job and a bit easier to read.
If `document` or `window` are not defined in the browser then there are big problems going on and likely other thing will break too.